todoman/model.py

```diff
@@ -585,8 +586,12 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             q = ', '.join(['?'] * len(lists))
             extra_where.append('AND files.list_name IN ({})'.format(q))
             params.extend(lists)
-        if urgent:
-            extra_where.append('AND priority = 9')
+        if priority is not None:
+            if priority == 0:
+                extra_where.append('AND priority = 0')
+            else:
+                extra_where.append('AND priority >0'
+                                   ' AND priority <= ' + priority)
```

Comment:
You are adding a string and an integer. I recommend using `"AND PRIORITY > 0 AND PRIORITY <= {}".format(priority)` instead.